projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/execute.py

```diff
@@ -50,6 +50,8 @@ def __model_executions(executions, output_format: OutputFormat) -> str:
         def transform_execution(e: DataJobExecution):
             d = e.to_dict()
             d["job_version"] = e.deployment.job_version
+            if e.deployment.python_version:
+                d["python_version"] = e.deployment.python_version
             del d["deployment"]
             return d
 
```

Comment:
nit: may add PR description of what's the change in the projects/vdk-control-cli/tests/resources/job-zip/test-job-python-version.zip ?

Reply:
Updated the PR description.